draft-dns-zone-digest.xml

```diff
@@ -538,6 +538,15 @@ An alternate method (rfc include) is described in the references. --><!ENTITY RF
           <t>
             The Digest field is a variable-length sequence of octets
             containing the output of the hash algorithm.
+          </t>
+          <t>
+          The current hashes for the SIMPLE scheme all require production and
+          inclusion of the entire length of the hash. Future hashes MAY be
+          truncated, but MUST not be truncated to a length that is less than
+          an equivalent to 96 bits of strength - e.g. 12 bytes for the SHA 2
+          and 3 family of hashes.
+          </t>
+          <t>
```

Comment:
I don't want to use the phrase "hashes for the SIMPLE scheme" because they are independent.  e.g. there might be a future scheme that also uses SHA384 hashing.  See branch Clarification_on_trunc_alt for my proposal.